.eslintrc.js

```diff
@@ -4,85 +4,57 @@ module.exports = {
     es2021: true,
   },
   settings: {
-    'import/resolver': {
+    "import/resolver": {
```

Comment:
I don't understand why all of these changes were committed. Nevertheless, it doesn't break anything.